pkgs/by-name/va/vapoursynth-bestsource/package.nix

```
@@ -0,0 +1,56 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  meson,
+  ninja,
+  cmake,
+  pkg-config,
+  vapoursynth,
+  ffmpeg,
+  xxHash,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "vapoursynth-bestsource";
+  version = "R6";
+
+  outputs = [
+    "out"
+    "dev"
+  ];
+
+  src = fetchFromGitHub {
+    fetchSubmodules = true;
+    owner = "vapoursynth";
+    repo = "bestsource";
+    rev = "refs/tags/${finalAttrs.version}";
+    hash = "sha256-ICkdIomlkHUdK6kMeui45fvUn4OMxSrP8svB2IN+GCg=";
+  };
```

Oh, the `R` shouldn't be feature in the derivation, only within the tag. Not sure if version should be set to `6` or `6.0.0` though

```suggestion
  version = "6";

  outputs = [
    "out"
    "dev"
  ];

  src = fetchFromGitHub {
    fetchSubmodules = true;
    owner = "vapoursynth";
    repo = "bestsource";
    rev = "refs/tags/R${finalAttrs.version}";
    hash = "sha256-ICkdIomlkHUdK6kMeui45fvUn4OMxSrP8svB2IN+GCg=";
  };
```